bayesian/utility/v2/ca_response_builder.py

```diff
@@ -472,24 +472,34 @@ def get_premium_message(self, exploitable_vuls: int) -> str:
         if self.public_vul and self.pvt_vul:
             # Add Private Vulnerability and Public Vul Info only
             message += f"{self.public_vul} known security vulnerability "
-            message += f"and {self.pvt_vul} security advisory "
-            message += f"with {exploitable_vuls} exploitable vulnerabilities. "
+            message += f"and {self.pvt_vul} security advisory with "
+            message += self.append_with_exploit_details(exploitable_vuls)
+            message += f"{self.severity[0]} severity. "
             message += self.get_recommendation()
             return message
 
         if self.public_vul:
             # Add Public Vulnerability Info only
-            message += f"{self.public_vul} known security vulnerability"
-            message += f" with {exploitable_vuls} exploitable vulnerabilities. "
+            message += f"{self.public_vul} known security vulnerability with "
+            message += self.append_with_exploit_details(exploitable_vuls)
+            message += f"{self.severity[0]} severity. "
```

Comment:
Is this 'high severity' text is extra in new message, it was not there before your changes?

Reply:
Yes, it is extra and it should be there: 
Jira story: https://issues.redhat.com/browse/APPAI-1512